specs/rails/bundle_install.yaml

```diff
@@ -0,0 +1,11 @@
+---
+name: Use bundler to install al gems
```

Comment:
Could we include "Rails" in the title of this workflow to make it more specific to the command? 

Something like:
`Use Rails bundle to install all gems`